### DISCUSSION TITLE
[求助]创建实例时，数据库类型没有clickhouse

### DISCUSSION THREAD

**Discussion by huskarz** [hhyo/Archery#2242] [Q&A]
docker compose 1.9.1

如图所示
![图片](https://github.com/hhyo/Archery/assets/5135906/61be1096-6ec8-4111-a11d-6b14c4f60df4)

**Comment by LeoQuote** [comment]
master版本才有，你换个tag试试

**Comment by huskarz** [comment]
好的，谢谢，已解决
